bundle/config/mutator/python/python_mutator.go

```diff
@@ -224,9 +225,10 @@ func (m *pythonMutator) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagno
 			return dyn.InvalidValue, mutateDiagsHasError
```

Comment:
1) Are we losing diags here?
2) Can we have more descriptive error than `errors.New("unexpected error")`

Reply:
No, we don't loose diags, I think we can improve the code, but let's do it separately it's completely unrelated to this change.